### PR TITLE
Add tvOS support

### PR DESCRIPTION
**Pull request by MongoCaleb**
### Jira

https://jira.mongodb.org/browse/DOCSP-27760

### Staged Changes
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/tvos/sdk/dotnet/compatibility/

### CODE REVIEW

**Review by krollins-mdb — APPROVED** [review]

File: source/sdk/dotnet/compatibility.txt [hunk 1]
```diff
@@ -50,6 +50,11 @@ version(s) you can use with Realm on your target platforms:
    * - Android
      - | Xamarin.Android for Android 4.1 (API level 16) or later. 
        | MAUI requires Android 5.0 (API level 21) or later.
+  
+   * - tvOS
+     - | Xamarin and Unity for tvOS 9.0 or later. 
+       | MAUI does not support tvOS.
```
Comment (nirinchev) [inline source/sdk/dotnet/compatibility.txt:56]: We don't need to explicitly call that out - e.g. we don't do that for Linux or UWP.

File: source/sdk/dotnet/compatibility.txt [hunk 2]
```diff
@@ -50,6 +50,11 @@ version(s) you can use with Realm on your target platforms:
...
+       | For more information on developing for tvOS, see :ref:`<ios-tvos>`.
```
Comment (nirinchev) [inline source/sdk/dotnet/compatibility.txt:57]: This links to the swift page - not sure if we need to recreate it for .NET, but there's a swift code snippet at the bottom, which can be rewritten in C#.

**Review by nirinchev — COMMENTED** [review]
A few minor comments. Also, completely unrelated to this page, but I saw the supported versions for desktop platforms say ".NET Core 2.0 or later (.NET 5.0 or later)", which is kind of weird - i.e. .NET 5.0 would be implicitly included in the "or later" part of .NET Core 2.0 - any reason we're calling it out explicitly?

File: source/sdk/dotnet/compatibility.txt (hunk 2 above)
Comment (MongoCaleb) [reply source/sdk/dotnet/compatibility.txt:57]: I think for now we won't recreate it, unless tvOS support becomes broad enough that we have people asking for code examples.

File: source/sdk/dotnet/compatibility.txt (hunk 1 above)
Comment (MongoCaleb) [reply source/sdk/dotnet/compatibility.txt:56]: @nirinchev  do we no longer need to differentiate between the .NET framework and .NET Core, which have different versioning schemes?

**Comment by MongoCaleb** [comment]
@nirinchev do we no longer need to differentiate between the .NET framework and .NET Core, which have different versioning schemes?

File: source/sdk/dotnet/compatibility.txt (hunk 2 above)
Comment (nirinchev) [reply source/sdk/dotnet/compatibility.txt:57]: Sounds good!

**Comment by nirinchev** [comment]
I think we should differentiate between .NET Framework and .NET Core, but not between .NET Core and .NET (i.e. the implication is that anything that supports .NET Core 2.0 supports .NET 5.0 and later).

**Pull request merged**